Amazon.IonObjectMapper/NamingConvention/SnakeCaseNamingConvention.cs

```diff
@@ -13,8 +13,18 @@
 
 namespace Amazon.IonObjectMapper
 {
+    /// <summary>
+    /// Snake Case naming convention for property names.
+    /// </summary>
     public class SnakeCaseNamingConvention : IIonPropertyNamingConvention
     {
+        /// <summary>
```

Comment:
Is this correct/ incorrect?

The camel case docstrings mismatch with this

Reply:
Addressed.